osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -228,6 +228,7 @@ protected override bool OnExiting(Screen next)
         {
             Content.ScaleTo(0.7f, 150, Easing.InQuint);
             this.FadeOut(150);
+            Background?.FadeColour(Color4.White, BACKGROUND_FADE_DURATION, Easing.OutQuint);
```

Comment:
It seems like in the breaking commit, `SongSelect.UpdateBeatmap()` wasn't updated to use `.FadeColour()` and currently still `.FadeTo()`s. That would be the appropriate place to fix this.